kpatch-build/kpatch-build

```diff
@@ -1238,6 +1284,19 @@ for i in $FILES; do
 			KOBJFILE_PATH="$OOT_MODULE"
 			SYMTAB="${TEMPDIR}/module/${KOBJFILE_NAME}.symtab"
 			SYMVERS_FILE="$TEMPDIR/Module.symvers"
+		elif [[ -n "$CONFIG_LTO_CLANG" ]] ; then
+			if [[ $KOBJFILE = vmlinux ]] ; then
+				KOBJFILE_NAME=vmlinux
+				KOBJFILE_PATH="$VMLINUX"
+				SYMTAB="${TEMPDIR}/${KOBJFILE_NAME}.symtab"
+				SYMVERS_FILE="$BUILDDIR/Module.symvers"
+			else
+				KOBJFILE_NAME=$(basename "${KOBJFILE%.ko}")
+				KOBJFILE_NAME="${KOBJFILE_NAME//-/_}"
+				KOBJFILE_PATH="${TEMPDIR}/module/$KOBJFILE.ko"
+				SYMTAB="${KOBJFILE_PATH}.symtab"
+				SYMVERS_FILE="$BUILDDIR/Module.symvers"
+			fi
```

Comment:
This change is almost completely copy-pasted from existing code and could surely be simplified?